docs/WinRTAPIContracts.md

```diff
@@ -0,0 +1,76 @@
+# Windows App SDK API Contracts
+
+## Summary
+Customers trust Microsoft that the products we make are sound, and will work.  Windows App SDK (WASDK) has recently released version 1.0.  As WASDK evolvs and more developers use WASDK in their own projects WASDK needs a way to guarantee that the API’s that ship are stable and that API versions are well documented.
```

Comment:
```suggestion
Customers trust Microsoft that the products we make are sound, and will work.  Windows App SDK (WASDK) has recently released version 1.0.  As WASDK evolves and more developers use WASDK in their own projects, WASDK needs a way to guarantee that the APIs that ship are stable and that API versions are well documented.
```

Document throughout needs a few spelling/grammar passes. Some nits that apply globally:
* Style nit: Remove double-space after full stop `.` (L14)
* Consistency nit: Remove `.` (L15) or add `.` to bulleted items (create global consistency)

Reply:
Never have I ever thought I'd be called out for adding two spaces after a period.

Reply:
>  add . to bulleted items (create global consistency)

Can you please explain?  All the lists are numbered.  Is there a list somewhere else that I can't see?